src/veterinary/Vet.java

```diff
@@ -4,20 +4,22 @@
 import animals.Animal;
 
 public class Vet extends Employee {
+	/**
```

Comment:
double start is reserved for functions and classes (i.e. "/**")

Reply:
fixed